prototypes/form-wizard/request.html

```diff
@@ -44,19 +44,24 @@
   <fieldset>
     <legend>Processing details</legend>
 
-    <label for="agency">Agency name</label>
-    <select id="agency">
-      <option value="usda">Department of Agriculture</option>
-      <option value="doj">Department of Justice</option>
-      <option value="gsa">General Services Administration</option>
-      <option value="i-dont-know">I don’t know</option>
-    </select>
-
-    <label for="component">Sub-agency/office/department</label>
-    <select id="component">
-      <option value="front-office">Front office</option>
-      <option value="i-dont-know">I don’t know</option>
-    </select>
+    <label for="agency" class="usa-input-required">Agency name</label>
+    <input id="agency" type="text" size="100">
+    <script>
+      jQuery(document).ready(function() {
+        $agency = jQuery('#agency');
+        $agency.typeahead({
+          minLength: 2,
+          highlight: true
+        }, {
+          limit: 10,
+          source: new Bloodhound({
+            local: FOIA.agencyNames(),
+            queryTokenizer: Bloodhound.tokenizers.whitespace,
+            datumTokenizer: Bloodhound.tokenizers.whitespace
+          })
+        });
+      });
+    </script>
```

Comment:
I would stick this in foia.js or something to avoid having it as an inline script.